docs/the-build-pipeline.md

```diff
@@ -17,11 +17,3 @@ However, in essence, here is how the build pipeline works:
 1. The documentation is built.
 1. The HTML version of the documentation is deployed.
 1. A build notification is sent.
-
-## Build Notifications
-
-Build notifications are sent to the `#documentation` channel in https://talk.owncloud.com when a full docs build fails. Notifications are not sent for successful builds.
```

Comment:
I think that the drone CI is currently doing this
https://github.com/owncloud/docs/blob/master/.drone.star#L165
```
"webhook": from_secret("rocketchat_talk_webhook")
```

I'm not 100% sure where that ends up these days. It might need to be adjusted or removed?